metricbeat/helper/server/http/http.go

```diff
@@ -107,17 +109,17 @@ func NewHttpServerWithHandler(mb mb.BaseMetricSet, handlerFunc http.HandlerFunc)
 func (h *HttpServer) Start() error {
 	go func() {
 		if h.server.TLSConfig != nil {
-			logp.Info("Starting HTTPS server on %s", h.server.Addr)
+			h.logger.Infof("Starting HTTPS server on %s", h.server.Addr)
 			//certificate is already loaded. That's why the parameters are empty
 			err := h.server.ListenAndServeTLS("", "")
 			if err != nil && err != http.ErrServerClosed {
-				logp.Critical("Unable to start HTTPS server due to error: %v", err)
+				h.logger.Errorf("Unable to start HTTPS server due to error: %v", err)
```

Comment:
Note for other reviewers: `Critical` is just an alias to `Error` and it's deprecated, see https://github.com/elastic/elastic-agent-libs/blob/020f7eb19e7f1451b94a5589e81adf653af66645/logp/global.go#L81-L86